test/arm/neon/st1_x4.c

```diff
@@ -0,0 +1,998 @@
+          { SIMDE_FLOAT32_C(   181.30), SIMDE_FLOAT32_C(  -312.67) },
+          { SIMDE_FLOAT32_C(   733.21), SIMDE_FLOAT32_C(   292.43) } },
+      { SIMDE_FLOAT32_C(  -655.57), SIMDE_FLOAT32_C(  -620.90), SIMDE_FLOAT32_C(  -517.88), SIMDE_FLOAT32_C(  -603.98),
+        SIMDE_FLOAT32_C(   181.30), SIMDE_FLOAT32_C(  -312.67), SIMDE_FLOAT32_C(   733.21), SIMDE_FLOAT32_C(   292.43) } },
+    {
+        { { SIMDE_FLOAT32_C(  -604.20), SIMDE_FLOAT32_C(   955.36) },
+          { SIMDE_FLOAT32_C(   492.52), SIMDE_FLOAT32_C(   660.31) },
+          { SIMDE_FLOAT32_C(   975.31), SIMDE_FLOAT32_C(   315.60) },
+          { SIMDE_FLOAT32_C(  -917.81), SIMDE_FLOAT32_C(  -330.72) } },
+      { SIMDE_FLOAT32_C(  -604.20), SIMDE_FLOAT32_C(   955.36), SIMDE_FLOAT32_C(   492.52), SIMDE_FLOAT32_C(   660.31),
+        SIMDE_FLOAT32_C(   975.31), SIMDE_FLOAT32_C(   315.60), SIMDE_FLOAT32_C(  -917.81), SIMDE_FLOAT32_C(  -330.72) } },
+    {
+        { { SIMDE_FLOAT32_C(    26.18), SIMDE_FLOAT32_C(  -986.00) },
+          { SIMDE_FLOAT32_C(  -306.20), SIMDE_FLOAT32_C(   428.83) },
+          { SIMDE_FLOAT32_C(  -783.34), SIMDE_FLOAT32_C(   141.79) },
+          { SIMDE_FLOAT32_C(   377.80), SIMDE_FLOAT32_C(   708.09) } },
+      { SIMDE_FLOAT32_C(    26.18), SIMDE_FLOAT32_C(  -986.00), SIMDE_FLOAT32_C(  -306.20), SIMDE_FLOAT32_C(   428.83),
+        SIMDE_FLOAT32_C(  -783.34), SIMDE_FLOAT32_C(   141.79), SIMDE_FLOAT32_C(   377.80), SIMDE_FLOAT32_C(   708.09) } },
+    {
+        { { SIMDE_FLOAT32_C(    44.11), SIMDE_FLOAT32_C(    29.43) },
+          { SIMDE_FLOAT32_C(  -379.13), SIMDE_FLOAT32_C(   547.87) },
+          { SIMDE_FLOAT32_C(   561.12), SIMDE_FLOAT32_C(    29.96) },
+          { SIMDE_FLOAT32_C(  -156.99), SIMDE_FLOAT32_C(   905.55) } },
+      { SIMDE_FLOAT32_C(    44.11), SIMDE_FLOAT32_C(    29.43), SIMDE_FLOAT32_C(  -379.13), SIMDE_FLOAT32_C(   547.87),
+        SIMDE_FLOAT32_C(   561.12), SIMDE_FLOAT32_C(    29.96), SIMDE_FLOAT32_C(  -156.99), SIMDE_FLOAT32_C(   905.55) } },
+    {
+        { { SIMDE_FLOAT32_C(   409.06), SIMDE_FLOAT32_C(   325.12) },
+          { SIMDE_FLOAT32_C(  -698.43), SIMDE_FLOAT32_C(  -409.64) },
+          { SIMDE_FLOAT32_C(  -987.55), SIMDE_FLOAT32_C(  -965.21) },
+          { SIMDE_FLOAT32_C(   882.79), SIMDE_FLOAT32_C(  -591.76) } },
+      { SIMDE_FLOAT32_C(   409.06), SIMDE_FLOAT32_C(   325.12), SIMDE_FLOAT32_C(  -698.43), SIMDE_FLOAT32_C(  -409.64),
+        SIMDE_FLOAT32_C(  -987.55), SIMDE_FLOAT32_C(  -965.21), SIMDE_FLOAT32_C(   882.79), SIMDE_FLOAT32_C(  -591.76) } },
+  };
+
+  for (size_t i = 0 ; i < (sizeof(test_vec) / sizeof(test_vec[0])) ; i++) {
+    simde_float32x2x4_t r_ = { { simde_vld1_f32(test_vec[i].r[0]),
+                                 simde_vld1_f32(test_vec[i].r[1]),
+                                 simde_vld1_f32(test_vec[i].r[2]),
+                                 simde_vld1_f32(test_vec[i].r[3]) } };
+
+    simde_float32 a_[8];
```

Comment:
Maybe this would fix [the bus error](https://github.com/simd-everywhere/simde/actions/runs/6557133868/job/17813232323?pr=1082#step:10:911)? 

```suggestion
    SIMDE_ALIGN_TO_16 simde_float32 a_[8];
```

If it works, apply this fix to the other declarations in the new tests of this PR.

Reply:
Sure, I'll give it a try.

Reply:
It works and has been applied to this PR.